Content.Client/State/LobbyState.cs

```diff
@@ -133,7 +135,7 @@ public override void FrameUpdate(FrameEventArgs e)
             }
             else
             {
-                var difference = _clientGameTicker.StartTime - DateTime.UtcNow;
+                var difference = _gameTiming.CurTime; 
```

Comment:
This is incorrect. This was supposed to be a difference of time between `StartTime` and the current time. Now, you set it to just the current time.